overlord/devicestate/handlers.go

```diff
@@ -581,17 +592,24 @@ func (m *DeviceManager) doRequestSerial(t *state.Task, _ *tomb.Tomb) error {
 		return fmt.Errorf("internal error: multiple serial assertions for the same device key")
 	}
 
-	serial, err := getSerial(t, privKey, device)
+	var serial *asserts.Serial
+	timings.Run(perfTimings, "get-serial", "get device serial", func(timings.Measurer) {
+		serial, err = getSerial(t, privKey, device)
```

Comment:
maybe we want to go a level deeper and measure prepare* and submit*

Reply:
Done.